ocaml/xenopsd/xc/device.ml

```diff
@@ -2082,7 +2082,7 @@ module Dm_Common = struct
          case. Don't pass -vgpu for a compute vGPU. *)
       match x with
       | Vgpu ({implementation= Nvidia {vclass= Some "Compute"; _}; _} :: _) ->
-          [] (* don't pass flags for VCS (compute) vGPU *)
+          ["-std-vga"]
```

Comment:
The commit message says `-vga` but the code says `-std-vga`. Which one is correct?

Reply:
I've updated it. The code is correct.